System/System.cpp

```diff
@@ -292,4 +314,69 @@ namespace RTE {
 			return (std::search(rawData.begin(), rawData.end(), findString.begin(), findString.end()) != rawData.end()) ? 0 : 1;
 		}
 	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+#ifdef __unix__
+	std::filesystem::path System::GetXdgDataHome() {
+		char *envXdgDataHome = std::getenv("XDG_DATA_HOME");
+		if (envXdgDataHome) {
+			return std::filesystem::path(envXdgDataHome) / std::filesystem::path("Cortex Command");
+		} else {
+			char *envUserHome = std::getenv("HOME");
+			if (!envUserHome) {
+				struct passwd *pw = getpwuid(getuid());
+				envUserHome = pw->pw_dir;
+			}
+			return envUserHome / std::filesystem::path(".local/share/Cortex Command/");
+		}
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void System::SetupBaseGameFolders() {
```

Comment:
Unless I'm mistaken, if Scenes.rte and Metagames.rte were included in the data repo, you'd be able to get rid of all this right?
I've been thinking as of late that it should be fine to include them there by default. If Data objected he's had years and several prepackaged builds to complain, so I'd say it's fair to get rid of that policy.

Reply:
~~Sadly no; anything that needs to be user writeable cannot be installed to system directories (readonly).~~ I just realized this can all can all be done with a simple bash script and 1~2 additional commandline parameters, that would also drastically simplify the install process

Reply:
Though if left in code, this already accounts for Scenes and Metagames being with the base data. Currently just generating those at build time

Reply:
Ahhh cause non-portable build keeps stuff in some system location, but has to keep these folder elsewhere so they're user-writable without sudo I guess? Or am I misunderstanding this work? I didn't look too closely at the end results haha

But yeah, if it won't help or hinder you to have the missing folders in the data repo that's good, I'll probably chuck em in sometime after pre4.